packages/compass-query-history/src/utils/query-storage.js

```diff
@@ -0,0 +1,59 @@
+import { promisifyAmpersandMethod } from 'mongodb-data-service';
+import { FavoriteQueryCollection, FavoriteQuery } from '../models';
+
+export class QueryStorage {
```

Comment:
suggestion: If this is only for favorites should we name it `FavoriteQueryStorage`?

Alternatively, if we feel we would have a `QueryStorage` for both favorites and history, we may want to rename the methods to `loadAllFavoriteQueries`, `updateFavoriteQueryAttributes` and `deleteFavoriteQuery`.

Reply:
renamed class to `FavoriteQueryStorage`